ceph.spec.in

```diff
@@ -536,6 +577,21 @@ librbd1 or libcephfs1 instead.
 
 %endif
 
+%package compat
```

Comment:
Hmm, ok, the patch does not delete the main ceph package. However, it deprecates (even obsoletes) it, here. I'm not sure what is the reason for that. The -compat packages are generally used to warn users that we are going to delete a (sub-)package. In this case, I don't think it would be of any benefit for us to delete the main ceph package. Why are we creating the ceph-compat package in the first place when the new ceph meta-package serves its purpose just as well?